sftp/1.3/modules/ROOT/edits/sftp-write.adoc

```diff
@@ -0,0 +1,75 @@
+= Write a File Using the SFTP Connector - Mule 4
+:page-aliases: connectors::sftp/sftp-write.adoc
+
+Anypoint Connector for SFTP (SFTP Connector) Write operation writes content into the given path on demand.
```

Comment:
Is there anything more to day about why/under what circumstances the audience wants or needs to do this? The benefit?

Reply:
Also, any way to work in mention of the subtopics?